src/sentry/sentry_apps/webhooks.py

```diff
@@ -0,0 +1,81 @@
+"""
+Webhook broadcasting utilities for Sentry applications.
+"""
+
+import logging
+from typing import Any
+
+from sentry.sentry_apps.logic import consolidate_events
+from sentry.sentry_apps.metrics import SentryAppEventType, SentryAppWebhookFailureReason
+from sentry.sentry_apps.services.app import app_service
+from sentry.sentry_apps.tasks.sentry_apps import send_webhooks
+from sentry.sentry_apps.utils.errors import SentryAppSentryError
+
+logger = logging.getLogger(__name__)
+
+
+def broadcast_webhooks_for_organization(
+    *,
+    event_type: str,
```

Comment:
Splitting this into 2 pieces (Resource Name & Event Name) would clean up some of the later logic I think.